### PR TITLE
docs(contributing): Write about Docker, Cube.js Server

### PR DESCRIPTION
**Pull request by ovr**
Hello!

Thanks

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -25,31 +25,62 @@ By contributing to Cube Dev, Inc., You accept and agree to the terms and conditi
 Cube.js works with Node.js 8+ and uses yarn as a package manager.
 
 ## Development Workflow
+
+### Cube.js Docker
+
+Cube.js offers two different types of docker's image:
```
Comment (hassankhan) [inline CONTRIBUTING.md:31]:
```suggestion
Cube.js offers two different types of Docker image:
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -25,31 +25,62 @@ By contributing to Cube Dev, Inc., You accept and agree to the terms and conditi
...
 
-1. After cloning Cube.js repository run `$ yarn` in `packages/cubejs-client-core` and `packages/cubejs-client-react` to install dependencies.
+1. After cloning Cube.js repository run `$ yarn install` and `$ yarn lerna bootstrap` in root directory.
 2. Use `$ yarn link` to add these packages to link registry.
 3. Perform required code changes.
 4. Use `$ yarn build` in the repository root to build CommonJS and UMD modules.
 5. Use `$ yarn link @cubejs-client/core` and/or `$ yarn link @cubejs-client/react` in your project to test changes applied.
 6. Use `$ yarn test` where available to test your changes.
-7. Ensure commit CommonJS and UMD modules as part of your commit.
+7. Ensure to commit CommonJS and UMD modules as part of your commit.
```
Comment (hassankhan) [inline CONTRIBUTING.md:56]:
```suggestion
7. Ensure that any CommonJS and UMD modules are included as part of your commit.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -25,31 +25,62 @@ By contributing to Cube Dev, Inc., You accept and agree to the terms and conditi
...
 
 To get set up quickly, you can perform 1) and 2) with one line from the `cube.js` clone root folder:
 
 ```
 cd packages/cubejs-client-core && yarn && yarn link && cd ../.. && cd packages/cubejs-client-react && yarn && yarn link && cd ../..
 ```
 
+### Cube.js Server
+
+Cube.js is written in plain JS, but some parts already migrated to TypeScript.
```
Comment (hassankhan) [inline CONTRIBUTING.md:66]:
```suggestion
Cube.js is written in plain JavaScript, but some parts have already been migrated to TypeScript.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -25,31 +25,62 @@ By contributing to Cube Dev, Inc., You accept and agree to the terms and conditi
...
+Cube.js offers two different types of docker's image:
+
+- Stable (building from published release on npm)
+- Dev (building from source files, needed to test unpublished changes)
+
+For more information, take a look at [Docker Development Guide](./packages/cubejs-docker/DEVELOPMENT.md).
+
+#### Stable Docker Release
+
+1. After cloning Cube.js repository run `$ yarn` in `packages/cubejs-docker` to install dependencies.
```
Comment (hassankhan) [inline CONTRIBUTING.md:40]: Can we remove the `$` from the beginning of all the inline commands, please?

**Review by hassankhan — CHANGES_REQUESTED** [review]
Awesome stuff @ovr 🙌 had some minor nitpicks, other than that looks great 👍

File: CONTRIBUTING.md (hunk 4 above)
Comment (ovr) [reply CONTRIBUTING.md:40]: I am using similar style across file, it was not added by me. 

cC @keydunov Is it ok to remove `$`?

Thanks

Comment (keydunov) [reply CONTRIBUTING.md:40]: Let's keep `$` for now as we're using it everywhere

**Comment by rpaik** [comment]
Thanks for this @ovr!

**Pull request merged**